app/views/hub/metrics/index.html.erb

```diff
@@ -8,7 +8,7 @@
       </div>
 
       <div class="spacing-above-15">
-        SLA Breach: Clients that haven't been contacted in 5 business days (since <%= long_formatted_datetime(@report.breached_at)%>)
+        SLA Breach: Clients that haven't been contacted in 6 business days (since <%= long_formatted_datetime(@report.breached_at)%>)
```

Comment:
Is this text change something product/design approves of? I guess it's an internal page anyway, but it's probably good to make sure they like whatever we end up with on this page. Not a blocker.

Reply:
The text change is accurate to the change they requested.